### PR TITLE
Fix capitalization/spacing in ArcGIS MapServer/FeatureServer strings

### PR DESCRIPTION
**Pull request by nyalldawson**
This should be "MapServer"/"FeatureServer", not "Map Server"/"Feature Server"

### CODE REVIEW

**Comment by nirvn** [comment]
For the record, I think its called ArcGIS Feature *Service*, not server (same goes for ArcGIS Map Service.

**Comment by nyalldawson** [comment]
> For the record, I think its called ArcGIS Feature Service, not server (same goes for ArcGIS Map Service.

I'm not sure. I think **maybe** the protocol is called "feature service", but an implementation of the protocol is a "feature server". Googling leads to a mix of the two terms used within the ArcGIS docs...

**Comment by nyalldawson** [comment]
Ok, so I think we should technically use "ArcGIS FeatureService Server". But that's incredibly clunky!

**Comment by nyalldawson** [comment]
Let's revise that: technically I think it should be "ArcGIS Server Feature Service", but then there IS 3rd party implementations of the protocol which should be supported in qgis already. So let's go with "ArcGIS Feature Service"...

**Comment by nirvn** [comment]
+1, you settled for my favorite option.

File: src/ui/qgisapp.ui [hunk 1]
```diff
@@ -2828,10 +2828,10 @@ Acts on the currently active layer only.</string>
      <normaloff>:/images/themes/default/mActionAddAfsLayer.svg</normaloff>:/images/themes/default/mActionAddAfsLayer.svg</iconset>
    </property>
    <property name="text">
-    <string>Add Ar&amp;cGIS FeatureServer Layer…</string>
+    <string>Add Ar&amp;cGIS Feature Service Layer…</string>
```
Comment (havatv) [inline src/ui/qgisapp.ui:2831]: What is the `&amp;` doing here?

Comment (DelazJ) [reply src/ui/qgisapp.ui:2831]: it's a shortcut

**Pull request merged**